app/src/main/kotlin/dev/soupslurpr/appverifier/InternalVerificationInfoDatabase.kt

```diff
@@ -1154,4 +1154,142 @@ val internalVerificationInfoDatabase = setOf(
             )
         )
     ),
+    InternalDatabaseVerificationInfo(
+        "io.simplelogin.android",
+        listOf(
+            // Google Play Store/GitHub
```

Comment:
The one on the Google Play Store has a different hash:
F2:02:30:63:37:BB:3B:C6:69:F7:E8:1D:10:2F:D1:10:C4:8C:67:2A:E4:56:3F:63:B9:7A:53:59:1D:F8:E8:CC

Why is yours the same as the GitHub one?